src/components/Speciality/BlogPosts.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
```

Comment:
renamed the old `blog.js` to `BlogPosts.js` since `blog` seems like it's a full blog, but we're really displaying some blog posts.